sunkit_instruments/io/__init__.py

```diff
@@ -0,0 +1 @@
+from .io import *  # NOQA
```

Comment:
This is a mistake on my part. 

I meant an io in `suvi`. So it would be `sunkit_instruments/suvi/io.py`

Reply:
Ok. I don't really know how to do that and tie it in with the rest of the suvi functions, so if you are unhappy with the way it is right now and you want it in the suvi subfolder instead of io, I need some detailed advice (along the lines of "put this here, import it in this file, write documentation here") on how to do that properly. I honestly don't understand how it is benefitial to put this one function in a different .py file, and then we hide it again by making it appear to all be part of one package. It just makes it unnecessarily complicated in my opinion, but what do I know about Python code development...

Reply:
My reasons are as follow:

It separates the logic of reading from other parts, for example, I could also ask to move the download function into a `download.py` (I won't but it is an example). 

This allows us to have shorter Python files that have a focus/goal as to what they contain. 
This makes it easier to browse and to edit files if we have to, long files are tricker to navigate. 
Furthermore, the separation of code makes testing simpler to deal with.

The reason I want a `suvi/io.py` over a overall `io/io.py` is that this IO is not general but is instrument dependent so it should be under the `suvi` folder.

Collapsing it all in `init.py` is done for the user and the overall style but internally we organize the code like this for the developers benefit. 

So overall I think the layout of the Python code would be:
```
suvi/__init__.py
suvi/io.py
suvi/suvi.py
suvi/tests/__init__.py
suvi/tests/test_io.py
suvi/tests/test_suvi.py
```

In the main `__init__.py` file, we will import from `suvi.py` and `io.py` so a user can import it all from one place.

Does this help in any way?

Reply:
Yes, it does. I don't necessarily agree with the reasoning, but I understand. I don't know yet when I will get to it and if I can integrate everything the way it is supposed to look like afterwards. I don't want to be snarky, I am not the managing the repo, and you probably have your reasons for wanting it this way. But try and look at it from my perspective: this is my first contribution, and the whole experience so far is not exactly encouraging me to contribute something else anytime soon. It's a huge time sink - I realize that it is for you too managing the repo, I am just trying to explain it.